horace_core/sqw/@sqw/mask_pixels.m

```diff
@@ -1,24 +1,24 @@
-function wout = mask_pixels (win, mask_array)
+function wout = mask_pixels (win, keep_obj)
 % Remove the pixels indicated by the mask array
 %
-%   >> wout = mask_pixels (win, mask_array)     % Mask array
+%   >> wout = mask_pixels (win, keep_obj)     % Mask array
 %   >> wout = mask_pixels (win, wmask)          % Mask according to pixel array
 %                                               % contents
 %
 % Input:
 % ------
 %   win                 Input sqw object
 %
-%   mask_array          Array of 1 or 0 (or true or false) that indicate
+%   keep_obj          Array of 1 or 0 (or true or false) that indicate
```

Comment:
maybe call it keep_pixels? Object sounds (a) vague (b) not a pixel

Reply:
according to the description, it may be `PixelData` object too. This option would certainly not work (for ticket), but naming needs to reflect this. Probably rename it to `keep_info`